docs/UserGuide.md

```diff
@@ -114,14 +154,12 @@ Format: <br>
  `add n/BOOK_TITLE g/GENRE [t/TAG]... tp/TOTAL_PAGES [b/PAGE_NUMBER]` <br>
 *tags and bookmarks are optional!*
 
-Examples:
-
 Example Command | Result
 -----------------|--------
 `add n/Fifty Shades of Gray g/Romance tp/350 b/200` | Adds a book named `Fifty Shades of Gray`, with genre `Romance`, with `350` total pages, bookmarked at page `200`
 `add n/Harry Potter and the Chamber of Secrets g/Fiction t/Wizards tp/1500 b/25` | Adds a book named `Harry Potter and the Chamber of Secrets`, with genre `Fiction`, with a tag `Wizards`, with `1500` total pages, bookmarked at page `25`
 
-### Viewing details of a book : `View`
+### Viewing details of a book : `view`
 
 Want a closer look? Look at the detailed information of your recorded book by using our 
```

Comment:
There is still no instructions on how to leave the detailed view, see #158 Could you add something like. "To go back to seeing all your books, use the [list command](#list-command)"

Reply:
Also maybe the part about  "any commands using `INDEX` can only refer to index 1 (the currently viewed book)" should be included in the view command itself instead? Can put it one of those styled divs: "After using the view command, the viewed book will have an `INDEX` of 1 since it is the only book shown on the screen. To interact with the book, use the `INDEX` of 1 to refer to the book"